src/main.py

```diff
@@ -0,0 +1,36 @@
+from scrape import get_style_data
+from utils import pretty_print, save_to_json
+
+import string
+import json
+
+from urllib.parse import quote
+from tqdm import tqdm
+
+def scrape_bjcp_website(output_file=None):
+    base_url = "https://www.bjcp.org/style/2021"
+    styles_data = []
+
+    for family_id in tqdm(range(1, 35), desc="Scraping families"):

```

Comment:
how do you know there are 34 families?

Reply:
It's a static site, it only updates once every couple of years.